src/idsk/components/footer-extended/template.njk

```diff
@@ -0,0 +1,131 @@
+{% from "details/macro.njk" import govukDetails %}
+<div class="idsk-footer__meta idsk-footer-extended {{ params.classes if params.classes }}">
+    <div class="govuk-width-container">
+        {% if params.firstTitle %}
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds two-thirds">
+                    <h3 class="govuk-heading-m ">
+                        {{ params.firstTitle }}
+                    </h3>
+                    <div class="govuk-grid-row small_subtitle">
+                        <div class="govuk-grid-column-one-half bottom-space">
+                            {% for i in range(0, params.numberOfSubtitles) -%}
+                                <li class="idsk-footer__inline-list-item">
+                                    <a class="idsk-footer__link" href="{{firstSubtitle.href}}">{{ params.firstSubtitle }}</a>
+                                </li>
+                            {%- endfor %}
+                        </div>
```

Comment:
tento "for" nie je dobry... to ti len vypisuje stale ten isty text: "params.firstSubtitle", ktomu {{firstSubtitle.href}} ani neexistuje cize nevypise nic... ak tomu chapem spravne, ma tu byt pole prvkov [{text: "xyz", link: "/xyz"}, ....] ktore sa maju vypisovat.. to iste vlastne plati pre vsetky "for"-i co tu mas